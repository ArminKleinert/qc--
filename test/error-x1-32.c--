const {
    true = 1 == 1;
}

foo() {
    return <3 / 0>;
}
