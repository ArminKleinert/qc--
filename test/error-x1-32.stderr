Character 42-57 Error: illegal <x/y> comnination
